cloudv2/counter-aws.w

```diff
@@ -9,7 +9,7 @@ pub interface IAwsCounter {
   dynamoTableName(): str;
 }
 
-pub class Counter_tfaws impl counter_types.ICounter, IAwsCounter {
+internal class Counter_tfaws impl counter_types.ICounter, IAwsCounter {
```

Comment:
Very cool with `internal` but I was actually wondering if it might be useful to expose these classes in case the user wants to explicitly instantiate an AWS counter (maybe under the `aws` namespace):

```js
bring cloud;

new cloud.aws.Counter();
```

Reply:
Yeah, I think that's probably a good call